unime/src/i18n/en/index.ts

```diff
@@ -243,6 +279,9 @@ const en = {
   CLOSE: 'Close',
   CONTINUE: 'Continue',
   SKIP: 'Skip',
+  REJECT: 'Reject',
+  ACCEPT: 'Accept',
+  APPROVE: 'Approve request',
```

Comment:
If it's a general approval, it could stay here. Since we changed it to a "Approve request", it's now specific and should be at `SCAN > SHARE_CREDENTIALS > APPROVE`.